encoding/box_query_encoding_test.go

```diff
@@ -0,0 +1,43 @@
+package encoding
+
+import (
+	"encoding/base64"
+	"github.com/algorand/go-algorand-sdk/client/v2/common/models"
+	"github.com/algorand/go-algorand-sdk/types"
+	"github.com/stretchr/testify/require"
+	"testing"
+)
+
+type example struct {
+	source           []byte
+	expectedEncoding string
+}
+
+var e = example{[]byte("tkÿÿ"), "b64:dGvDv8O/"}
+
+func TestEncode(t *testing.T) {
```

Comment:
Is this test run in the CI? I briefly checked the logs and it seems like only the cucumber tests are run. I'm not too familiar with the repo, but I'm guessing this could also be the case with all the other `*_test.go` files

Reply:
@algochoi To close the loop from our verbal - It's a bit _non_-obvious, but the non-Cucumber tests _are_ run by CI.

* The build runs `make unit` (https://github.com/algorand/go-algorand-sdk/blob/develop/Makefile#L20), which includes the non-Cucumber tests.
* Searching the full CI build output (https://circleci.com/api/v1.1/project/github/algorand/go-algorand-sdk/985/output/108/0?file=true&allocation-id=62c8753c4f212d3f212b6fba-0-build%2FDRTQJFDE) for `go test ` shows the tests running.
  * Unclear if by-design, but `make integration` also runs the non-Cucumber tests.  Explains why the output repeats.

Reply:
appreciate the clarification!